docs/ruleset_functions/get_proposed_hvac_modeled_with_virtual_heating.md

```diff
@@ -0,0 +1,54 @@
+ 
+**Function Call:**   
+
+1. match_data_element()  
+2. match_data_element_exist()
+
+## Logic:  
+- For each building_segment_p in the P_RMR...BuildingSegment: `for building_segment_p in P_RMR...BuildingSegment:`
+    - Get analogous segment in the U_RMR: `building_segment_u = match_data_element(U_RMR,BuildingSegment,building_segment_p.id)`
+    - Get list of hvac systems in the U_RMR: `hvac_list_u = building_segment_u.heating_ventilation_air_conditioning_systems`
+    - For each hvac_p in the building_segment_p: `for hvac_p in building_segment_p:`    
+        The line below will check if a virtual HVAC system was modeled in the proposed. If hvac_p is not in the U_RMR then its a virtual system but need to check if virtual heating or virtual cooling or both was modeled. 
+        - Check if the hvac system in the proposed design model is NOT in the U_RMR: `if hvac_p.id not in hvac_list_u:`
+            - Reset has_virtual_heating_p boolean variable: `has_virtual_heating_p = FALSE`   
+            - Check if there are any heating_system associated with hvac_p, if not skip to preheat check: `if hvac_p.heating_system != Null:`
+                Below checks if a heating system specifically was modeled (could just be cooling for this hvac system) for the vitual system in the proposed. 
+                - For each heating_system_p in hvac_p: `for heating_system_p in hvac_p.heating_system:`
+                    - Check if the heating_system_type != "None": `if heating_system_p.heating_system_type != "None":`
+                        - Set has_virtual_heating_p to true: `has_virtual_heating_p = TRUE`
+            - Check if there are any preheat_system associated with hvac_p, if not skip indented code: `if hvac_p.preheat_system != Null:`
+                Below checks if a preheating system specifically was modeled (could just be cooling for this hvac system) for the vitual system in the proposed. We are checking the preheat coils because baseline systems 5 through 8 have preheat coils. 
+                - For each preheat_system_p in hvac_p: `for preheat_system_p in hvac_p.preheat_system:`
+                    - Check if the preheat_system_type != "None": `if preheat_system_p.preheat_system_type != "None":`
+                        - Set has_virtual_heating_p to true: `has_virtual_heating_p = TRUE`              
+        - Else, the hvac system in the proposed design model is also in the U_RMR: `Else:`
+            - Reset has_virtual_heating_p boolean variable: `has_virtual_heating_p = FALSE`   
+            Below compares the heating systems associated with the hvac system across the P_RMR and U_RMR only if heating was modeled in the proposed. If heating was modeled in the proposed and NOT in the U_RMR (the heating system type is None in the U_RMR) then this means virtual heating was modeled in the proposed. We are checking the preheat coils because baseline systems 5 through 8 have preheat coils. 
```

Comment:
@claperle I understand the logic in the RDS, however each time I've reviewed this I have gotten confused since this is a complicated funtion.
Would it make sense ro simply include a summary in the beginnning which says something like

This RDS has 4 steps
1. When system in P is not present in U: Lines 22 to 29 check if a system in P RMR is not present in the U RMR. If the heating type != None, it sets virtual heating to TRUE.
2. When system in P is not present in U: Lines 30 to 34 check if P RMR has preheat which is not present in U RMR. If yes, its sets virtual heating to TRUE
3. When system in P is present in U: Lines 31 to 43 repeat Step 1 for systems in both P and U RMR.
4. When system in P is present in U: Lines 44 to 50 repeat Step 2 for systems in both P and U RMR.